pom.xml

```diff
@@ -1258,6 +1264,8 @@
                 launched by the tests have access to the correct test-time classpath.
               -->
               <SPARK_DIST_CLASSPATH>${test_classpath}</SPARK_DIST_CLASSPATH>
+              <SPARK_PREPEND_CLASSES>1</SPARK_PREPEND_CLASSES>
```

Comment:
So basically tests will run in 'prepend classes' mode all the time now instead of needing the assembly?